app/Http/Middleware/SecurityHeaders.php

```diff
@@ -16,7 +16,7 @@ class SecurityHeaders
      */
     public function handle($request, Closure $next)
     {
-        $allowedHosts = ['helpforhealth.local', 'impreunapentrusanatate.ro', 'dev.impreunapentrusanatate.ro'];
+        $allowedHosts = ['helpforhealth.loc', 'impreunapentrusanatate.ro', 'dev.impreunapentrusanatate.ro'];
```

Comment:
unrelated to task, please revert

```suggestion
        $allowedHosts = ['helpforhealth.local', 'impreunapentrusanatate.ro', 'dev.impreunapentrusanatate.ro'];
```

Reply:
my bad i used a diff . name
one  my ask qhy ask why we so that check when that check should be in CFG

Reply:
nu zic că trebuie să existe, ci doar că nu e legat de task-ul curent de localization